tensorflow/core/util/tensor_bundle/byte_swap_tensor.cc

```diff
@@ -161,6 +161,14 @@ absl::Status ByteSwapTensor(Tensor* t) {
                         t->NumElements());
 }
 
+absl::Status ByteSwapTensorProto(TensorProto* tp) {
+  std:string content_str = std::string(tp->tensor_content());
```

Comment:
```suggestion
  std::string content_str = std::string(tp->tensor_content());
```

Oh, this is a typo

Reply:
But it breaks internally as `std:` becomes an unused label and we compile with `-Werror`

Reply:
🤦 done